src/sentry/reprocessing2.py

```diff
@@ -202,16 +207,47 @@ def mark_event_reprocessed(data):
     _get_sync_redis_client().decr(key)
 
 
-def start_group_reprocessing(project_id, group_id, max_events=None):
-    from sentry.models.group import Group, GroupStatus
-    from sentry.models.grouphash import GroupHash
+def start_group_reprocessing(project_id, group_id, max_events=None, acting_user_id=None):
     from django.db import transaction
 
     with transaction.atomic():
```

Comment:
btw this might be a pretty large transaction, it touches a lot of tables at once.

Reply:
Let's follow up on this please. I'm pretty sure we'll want to back this out.

Reply:
The problem is that I would really not like to fail halfway through, because that's quite a mess to clean up. Perhaps we want some shitty kind of "clientside rollback", I think I saw that someplace else already.